### DISCUSSION TITLE
I can't understand how to run this line ./quantize ./models/7B/ggml-model-f16.bin ./models/7B/ggml-model-q4_0.bin 2

### DISCUSSION THREAD

**Discussion by SCHIZ0FRENIA** [ggml-org/llama.cpp#923] [Q&A]
I called it like this 
python3 ./quantize ./models/7B/ggml-model-f16.bin ./models/7B/ggml-model-q4_0.bin 2

Like this 
./quantize ./models/7B/ggml-model-f16.bin ./models/7B/ggml-model-q4_0.bin 2

but nothing works, please help me.

**Comment by heroofhyla** [comment]
The second one should be correct. What errors are you getting?
